### PR TITLE
Add Music

### PR DESCRIPTION
**Pull request by aurelenc**

### CODE REVIEW

File: src/game/Engine.hpp [hunk 1]
```diff
@@ -22,10 +22,17 @@
 #include "Users.hpp"
 #include "settings/Settings.hpp"
 
+#include "raylib/core/Music.hpp"
+
 namespace game
 {
     class Engine {
       public:
+        enum PreloadedMusics {
```
Comment (MisterPeModder) [inline src/game/Engine.hpp:31]: The plural of "music" is "music"

Comment (MisterPeModder) [reply src/game/Engine.hpp:31]: `PreloadedMusicTracks` you if want to keep the plural

File: src/raylib/core/Music.hpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    { /// Core namespace (inside of raylib)
+        class Music {
+          public:
+            /// Default constructor
+            Music();
+
+            // /// Consutructor of Music wich create the musics
+            // ///
+            // /// @param filename Music path
+            // Music(const std::filesystem::path &fileName);
```
Comment (MisterPeModder) [inline src/raylib/core/Music.hpp:32]: Don't leave commented-out code.

**Review by MisterPeModder — CHANGES_REQUESTED** [review]

**Review by AndreasLrx — CHANGES_REQUESTED** [review]

File: src/game/scenes/SettingsMenuScene.cpp [hunk 3]
```diff
@@ -312,7 +312,7 @@ static void loadAudioSettings(ecs::World &world, raylib::core::Vector2f pos, ray
                     world.getResource<game::resources::EngineResource>().engine->getSettings().setMasterVolume(0.f);
                     Logger::logger.log(Logger::Severity::Debug, "Volume set to 0%");
                 } else {
-                    raylib::core::Audio::setMasterVolume(100.f);
+                    raylib::core::Audio::setMasterVolume(1.f);
```
[the remark this answers is not in the document]
Comment (MisterPeModder) [reply src/game/scenes/SettingsMenuScene.cpp:315]: Come on @AndreasLrx, be kind to my poor ears...

File: src/raylib/core/Music.cpp [hunk 4]
```diff
@@ -0,0 +1,71 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Music
+*/
+
+#include "Music.hpp"
+#include <iostream>
```
[the remark this answers is not in the document]
Comment (MisterPeModder) [reply src/raylib/core/Music.cpp:9]: And he didn't even used the logger!

File: src/game/scenes/SettingsMenuScene.cpp (hunk 3 above)
[the remark this answers is not in the document]
Comment (AndreasLrx) [reply src/game/scenes/SettingsMenuScene.cpp:315]: Fine...

**Review by AndreasLrx — APPROVED** [review]

**Review by MisterPeModder — APPROVED** [review]

**Review by ZQUEMA — APPROVED** [review]

**Pull request merged**